src/renderer/src/pages/settings/settings-general.tsx

```diff
@@ -15,34 +18,41 @@ export function SettingsGeneral({
   userPreferences,
   updateUserPreferences,
 }: SettingsGeneralProps) {
+  const { t } = useTranslation("settings");
+
   const [form, setForm] = useState({
     downloadsPath: "",
     downloadNotificationsEnabled: false,
     repackUpdatesNotificationsEnabled: false,
+    language: "",
   });
 
+  const [defaultDownloadsPath, setdefaultDownloadsPath] = useState("");
```

Comment:
rename `setdefaultDownloadsPath` to `setDefaultDownloadsPath` to follow camelCase pattern